modules/upgrade-central-cluster-crds.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="upgrade-central-cluster-crds{context}"]
+= Upgrading `SecurityPolicy` CRD
```

Comment:
```suggestion
= Upgrading the SecurityPolicy custom resource definition
```
We can't have [backticks in module titles](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)